### PR TITLE
Allow blank organization.name_prefix.

### LINKED ISSUE #120: Make "Naam (voorvoegsel)" optional for organizations
Currently, "Naam (voorvoegsel)" is required for organization. While this is logical for departments within EEMCS, regular companies do not have a prefix. Should make this attribute optional.

### PR DESCRIPTION
**Pull request by svenvanhal**
Fixes #120.

### CODE REVIEW

**Comment by svenvanhal** [comment]
Hmm, branch name is incorrect, not a bugfix..

File: ldb/templates/ldb/organization_detail.html [hunk 1]
```diff
@@ -24,9 +24,9 @@ <h2 class="page-header">{{ organization }}
         <fieldset>
           <legend>Algemeen</legend>
           <div class="col-xs-12">
-            {{ organization.name_prefix }}<br/>
+            {% if organization.name_prefix %}{{ organization.name_prefix }}{% endif %}<br/>
```
Comment (praseodym) [inline ldb/templates/ldb/organization_detail.html]: This doesn't change anything does it?

Comment (jgadelange) [inline ldb/templates/ldb/organization_detail.html]: Nope

Comment (svenvanhal) [inline ldb/templates/ldb/organization_detail.html]: Ah, ```<br />``` should be inside the if statement

**Comment by praseodym** [comment]
Squash plx

**Comment by jgadelange** [comment]
:shipit:

**Pull request merged**